substrate/rpc/src/author/mod.rs

```diff
@@ -35,8 +37,11 @@ build_rpc_trait! {
 	/// Substrate authoring RPC API
 	pub trait AuthorApi<Hash, Extrinsic> {
 		/// Submit extrinsic for inclusion in block.
+		#[rpc(name = "author_submitRichExtrinsic")]
+		fn submit_rich_extrinsic(&self, Extrinsic) -> Result<Hash>;
```

Comment:
I'm not sure about naming scheme choosen here. I kinda understand why it is "rich" extrinsic, but maybe it is better to leave it as "submit_extrinsic" and make another procedure call "submit_raw_extrinsic"? 

Or are we constrained by backward compatibility here?

Reply:
most practical uses of substrate will be easier to use with the raw variant, so it makes sense to me to optimize the API a bit for it.